juju/python-libjuju
# [JUJU-4188] Add 2.9.43 facades

#### Description

Updates the facades with the latest schema from Juju 2.9 branch, regenerates the clients, and adds the `AgentLifeFlag` facade to the facades list.


#### QA Steps

Pretty straightforward console connection should suffice (as the `m.connect()` is doing a bunch of stuff with the facades under the hood to establish a connection):

```sh
$ python -m asyncio
asyncio REPL 3.10.6 (main, May 29 2023, 11:10:38) [GCC 11.3.0] on linux
Use "await" directly instead of "asyncio.run()".
Type "help", "copyright", "credits" or "license" for more information.
>>> import asyncio
>>> from juju import model;m = model.Model();await m.connect()
>>>
```

Without this change you'd see additional output saying:

```sh
unknown facade AgentLifeFlag
unexpected facade AgentLifeFlag found, unable to decipher version to use
```
